.travis.yml

```diff
@@ -1,11 +1,14 @@
+sudo: required
+dist: trusty
```

Comment:
@ascripcaru Hi, why are we locking build to some particular Ubuntu release here?